dvc/command/plot.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+import logging
+
+from dvc.command.base import append_doc_link, CmdBase
+from dvc.utils import format_link
+
+logger = logging.getLogger(__name__)
+
+
+class CmdPlot(CmdBase):
+    def run(self):
+        self.repo.plot(self.args.targets)
```

Comment:
Nitpick: It could raise an exception from somewhere, right? Let's wrap it into try&except, same as we do in other places.